README.md

```diff
@@ -24,68 +27,106 @@ We like to think of it as `kubectl` for clusters.
   </a>
 </p>
 
-## Launching a Kubernetes cluster hosted on AWS
+
+## Launching a Kubernetes cluster hosted on AWS or GCE
 
 To replicate the above demo, check out our [tutorial](/docs/aws.md) for
 launching a Kubernetes cluster hosted on AWS.
 
+To install a Kuberentes cluster on GCE please follow this [guide](/docs/tutorial/gce.md).
+
 
 ## Features
 
-* Automates the provisioning of Kubernetes clusters in ([AWS](/docs/aws.md))
+* Automates the provisioning of Kubernetes clusters in [AWS](/docs/aws.md) and [GCE](/docs/tutorial/gce.md)
 * Deploys Highly Available (HA) Kubernetes Masters
-* Supports upgrading from [kube-up](/docs/upgrade_from_kubeup.md)
 * Built on a state-sync model for **dry-runs** and automatic **idempotency**
-* Ability to generate configuration files for AWS [CloudFormation](https://aws.amazon.com/cloudformation/) and Terraform [Terraform configuration](/docs/terraform.md)
+* Ability to generate [Terraform](/docs/terraform.md)
 * Supports custom Kubernetes [add-ons](/docs/addons.md)
 * Command line [autocompletion](/docs/cli/kops_completion.md)
-* Manifest Based API [Configuration](/docs/manifests_and_customizing_via_api.md)
-* Community supported!
+* YAML Manifest Based API [Configuration](/docs/manifests_and_customizing_via_api.md)
+* [Templating](/docs/cluster_template.md) and dry-run modes for creating
+ Manifests
+* Choose from eight different CNI [Networking](/docs/networking) providers out-of-the-box
+* Supports upgrading from [kube-up](/docs/upgrade_from_kubeup.md)
+* Capability to add containers, as hooks, and files to nodes via cluster manifest
 
 
-## Documentations
+## Documentation
 
 Documentation is in the `/docs` directory, [and the index is here.](docs/README.md)
 
 
+## Kubernetes Release Compatibility
+
+`kops` is intended to be backward compatible.  It is always recommended to use
+the latest version of kops with whatever version of Kubernetes you are using.
+`kops` supports the matching Kubernetes minor release.   For instance kops,
+1.8.x does not support Kubernetes 1.9.x, but kops 1.9.x  supports Kubernetes
+1.9.x.  Moreover, kops 1.8.x attempts to be backward compatible and support
```

Comment:
This should say "but kops 1.9.x would support kubernetes 1.8.x" right?

Reply:
👍